Ktisis/Interface/Input.cs

```diff
@@ -253,5 +279,44 @@ public enum Purpose {
 			}).ToDictionary(kp => kp.purpose, kp => kp.state);
 		}
 		private unsafe static bool IsChatInputActive() => ((UIModule*)Services.GameGui.GetUIModule())->GetRaptureAtkModule()->AtkModule.IsTextInputActive() == 1;
+
+		private static readonly List<string> MouseObstructingAddonNames = new() {
+				{ "CameraSetting"},
+				{ "ChatLog"},
+				{ "ChatLogPanel_0" },
+				{ "ChatLogPanel_1" },
+				{ "ChatLogPanel_2" },
+				{ "ChatLogPanel_3" },
+				{ "GroupPoseGuide" },
+				{ "GroupPoseStamp" },
+				// { "GroupPoseStampImage" }, // this seem to take almost all the screen
+			};
+		private unsafe static bool IsMouseOverWindow() {
+
+			foreach(var addonName in MouseObstructingAddonNames) {
+				var address = Services.GameGui.GetAddonByName(addonName, 1);
+				if (address == IntPtr.Zero) continue;
+
+				var addon = (FFXIVClientStructs.FFXIV.Component.GUI.AtkUnitBase*)address;
+				if (addon == null && !addon->IsVisible) continue;
```

Comment:
If the LHS of this `&&` is `true`, the RHS will result in a `NullReferenceException`. Did you mean this?
```suggestion
				if (addon == null || !addon->IsVisible) continue;
```